pkg/ddl/backfilling_dispatcher.go

```diff
@@ -336,42 +319,6 @@ func generateNonPartitionPlan(d *ddl, tblInfo *model.TableInfo, job *model.Job)
 	return subTaskMetas, nil
 }
 
```

Comment:
Will we need to make the subtask with more regions？

Reply:
Need to change comments in backfilling_proto.go

Reply:
https://github.com/pingcap/tidb/pull/47982/commits/3be9b60540ef6ec586d12a1bae5209d5b9fdd191

Reply:
https://github.com/pingcap/tidb/pull/47982/commits/3be9b60540ef6ec586d12a1bae5209d5b9fdd191